samples/subsys/power/stm32_pm/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  name: Low Power Timer Sample for nucleo board
+tests:
+  sample.power.pm.stm32_lptim.lsi:
+    platform_whitelist: nucleo_l4r5zi
```

Comment:
can we try to remove withelist and filter on LTPIM support ? 
This would enable testing on more platforms

Reply:
no, removed